test/e2e/specs/nux.test.js

```diff
@@ -2,13 +2,34 @@
  * Internal dependencies
  */
 import '../support/bootstrap';
-import { newDesktopBrowserPage, newPost } from '../support/utils';
+import {
+	clearLocalStorage,
+	clickOnMoreMenuItem,
+	newDesktopBrowserPage,
+	newPost,
+} from '../support/utils';
 
-describe( 'NUX', () => {
-	it( 'should show tips to a first-time user', async () => {
+describe( 'New User Experience (NUX)', () => {
+	const NUMBER_OF_TIPS = 4;
+
+	async function getTipsLocalStorage( page ) {
+		return await page.evaluate( () => {
+			return JSON.parse( window.localStorage.GUTENBERG_NUX_1 );
+		} );
+	}
+
+	beforeEach( async () => {
 		await newDesktopBrowserPage();
 		await newPost( undefined, false );
+	} );
+
+	afterEach( async () => {
+		// Clear localStorage tips so they aren't persisted for the next test.
+		await clearLocalStorage();
+		await page.goto( 'about:blank' );
```

Comment:
What does this line do?

Reply:
When I was doing this in `beforeEach` I needed it for the tests to pass, I'm guessing because the page had loaded first. But come to think of it they might not be needed since it's moved to the `afterEach` section; I'll see about removing it.